Sources/BraintreePayPalNativeCheckout/BTPayPalNativeCheckoutClient.swift

```diff
@@ -71,6 +71,9 @@ import PayPalCheckout
                 )
 
                 PayPalCheckout.Checkout.set(config: payPalNativeConfig)
+                
+                NotificationCenter.default.post(name: Notification.Name("brain_tree_source_event"), object: nil)
```

Comment:
Should this be `braintree_source_event`? The company name is all one word or do we also separate out PayPal like `pay_pal` for these events?

Reply:
This string has already been merged in the SDK and a release is being cut. Will have to change it next cycle but this string is not used in reporting/recording. Only used as a key.